### PR TITLE
PL Release Notes 789xx

### PR DESCRIPTION
**Pull request by rashminandasahoo**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: release-notes/harness-platform.md [hunk 1]
```diff
@@ -17,6 +17,36 @@ Harness deploys updates progressively to different Harness SaaS clusters. You ca
...
+
+### Early access
+
+This release does not include any early access feature.
+
+### Fixed issues
+
+- JWT log sanitizer throws a null pointer exception when it receives null log messages. (PL-32136)
+
+  A code enhancement to add an empty check before sanitizing a log line for secret and JWT masking has fixed this issue.
```
Comment (SmritiSatya) [inline release-notes/harness-platform.md:39]: A code enhancement has fixed this issue, that is, an empty check has been added before a log line is sanitized for secrets and JWT masking.
Was trying to improve readability. Not sure if this is good.

**Review by SmritiSatya — DISMISSED** [review]

File: release-notes/harness-platform.md [hunk 2]
```diff
@@ -17,6 +17,36 @@ Harness deploys updates progressively to different Harness SaaS clusters. You ca
 Additionally, the release notes below are only for NextGen SaaS. FirstGen SaaS release notes are available [here](/docs/first-gen/firstgen-release-notes/harness-saa-s-release-notes) and Self-Managed Enterprise Edition release notes are available [here](/release-notes/self-managed-enterprise-edition).
 :::
 
+## March 31, 2023, version 789xx
+
+### What's new
+
+- The following APIs now return only the permitted service accounts: (PL-32080)
```
Comment (KapilDigani) [inline release-notes/harness-platform.md:24]: This was always the case - so no change in behaviour of the API here. What has changed is - earlier we didn't allow customers to pick specific service accounts in the Resource Groups  .. we now have added the functionality to allow customers to pick specific service accounts. What has changed is how Service Accounts are selected in the Resource Group not in the way API works from the customers' point of view

Comment (rashminandasahoo) [reply release-notes/harness-platform.md:24]: @KapilDigani Selection of specific service accounts has already been added to the release notes for version 78817.
If API has no change, then we should mark this as Release Note candidate `No'. Please advise.

File: release-notes/harness-platform.md [hunk 3]
```diff
@@ -17,6 +17,32 @@ Harness deploys updates progressively to different Harness SaaS clusters. You ca
 Additionally, the release notes below are only for NextGen SaaS. FirstGen SaaS release notes are available [here](/docs/first-gen/firstgen-release-notes/harness-saa-s-release-notes) and Self-Managed Enterprise Edition release notes are available [here](/release-notes/self-managed-enterprise-edition).
 :::
 
+## March 31, 2023, version 789xx
+
+### What's new
+
+- The favorite icon now dynamically changes based on pipeline execution status on the dashboard's execution view. (PL-31520)
```
Comment (SudarshanPurohit) [inline release-notes/harness-platform.md:24]: Let's leave this as favicon, that's what most people call it.

File: release-notes/whats-new.md [hunk 4]
```diff
@@ -16,6 +16,12 @@ Harness deploys updates progressively to different Harness SaaS clusters. You ca
 Additionally, the release notes below are only for NextGen SaaS. FirstGen SaaS release notes are available [here](/docs/first-gen/firstgen-release-notes/harness-saa-s-release-notes) and Self-Managed Enterprise Edition release notes are available [here](/release-notes/self-managed-enterprise-edition).
 :::
 
+## March 31, 2023, version 789xx
+
+### Harness Platform
+
+- The favorite icon now dynamically changes based on pipeline execution status on the dashboard's execution view. (PL-31520)
```
Comment (SudarshanPurohit) [inline release-notes/whats-new.md:23]: favicon

**Review by SudarshanPurohit — APPROVED** [review]

**Pull request merged**